solidity/test/lib/mailboxes.ts

```diff
@@ -18,8 +18,8 @@ import {
   LegacyMultisigIsm,
   TestMailbox,
   TestMerkleTreeHook,
-} from '../../types';
-import { DispatchEvent } from '../../types/contracts/Mailbox';
+} from '../../core-utils/evm/types';
```

Comment:
a bunch of tests still reference `core-utils/evm` but it looks like it's been moved instead to `core-utils/typechain`

Reply:
Fixed!